realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -313,7 +315,8 @@ private void emitPersistedFieldAccessors(final JavaWriter writer) throws IOExcep
             } else if (Utils.isRealmModel(field)) {
                 emitRealmModel(writer, field, fieldName, fieldTypeCanonicalName);
             } else if (Utils.isRealmList(field)) {
-                emitRealmList(writer, field, fieldName, fieldTypeCanonicalName);
+                final TypeMirror elementTypeMirror = Utils.getRealmListElementTypeMirror(field);
```

Comment:
Wouldn't this method be one that fit inside the `TypeMirrors` class?

Reply:
What do you mean? `TypeMirror` is not our class, but a part of Java framework.

Reply:
done